internal/cli/args.go

```diff
@@ -0,0 +1,80 @@
+package cli
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type argsCmd struct {
+	webPort int
+	clear   bool
+	post    httpPoster
+}
+
+func newArgsCmd() *argsCmd {
+	return &argsCmd{post: http.Post}
+}
+
+func (c *argsCmd) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "args [<tiltfile args>]",
+		Short: "Changes the Tiltfile args in use by a running Tilt",
```

Comment:
maybe slightly more correct to say "for a running Tilt _instance_"? (I... think?) if you change it here, also change in the description for "port" below

Reply:
> This is a nit and I'm not sure how important it is to clarify for users, but Tilt doesn't run on a port, the webserver runs on a port. (also, i feel like this implementation isn't compatible with someone running Tilt with `--port=0`, is that important?)

At the risk of getting arguey, I'm not sure I agree with the value or correctness of that distinction. If someone mentioned the port their docker was running on and someone else replied "docker doesn't run on a port, its HTTP API runs on a port", is that reducing confusion? I'm not even sure what "run on a port" would mean at that point. Maybe one could make an argument about the extent to which a program's interface is via the port.

I suspect that almost none of our users use --port, which would make discussion of that noise, and for the users who do use --port, they're presumably running like `tilt up --port 3248` and "running Tilt on a port" feels like a good match to that usage?

aaaand, I guess we're still logging "running on a port" to users who aren't specifying a --port, so that's not as clean as I thought. How about `fmt.Printf("changed config args for Tilt running at %s to %v\n", schemeHostPort, args)`?